docs/source/en/api/pipelines/latte.md

```diff
@@ -0,0 +1,75 @@
+<!-- # Copyright 2024 the Latte Team and The HuggingFace Team. All rights reserved.
```

Comment:
We usually only include Hugging Face for documentation.

Reply:
Okay. We can delete the Latte team.